ParisBNF/abb/BNFabb137.xml

```diff
@@ -0,0 +1,234 @@
+                     <note>The manuscript was possibly copied before <date>1849</date>
+                     since <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
+                     left Ethiopia in 1848.
+                     </note>
+                </origin>
+                <acquisition></acquisition>
+
+
+                    </history>
+                    <additional>
+                       <adminInfo>
+                          <recordHist>
+                             <source>
+                                <listBibl type="catalogue">
+                                   <bibl>
+                                      <ptr target="bm:ContiRossini1914Notice"/>
+                                      <citedRange unit="page">34-35</citedRange>
+                                      <citedRange unit="number">10</citedRange>
+                                      </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Chaine1912Catalogue"/>
+                                      <citedRange unit="page">86</citedRange>
+                                      <citedRange unit="number">137</citedRange>
+                                   </bibl>
+                                   <bibl>
+                                      <ptr target="bm:Abbadie1859Cat"/>
+                                      <citedRange unit="page">153</citedRange>
+                                      <citedRange unit="number">137</citedRange>
+                                   </bibl>
+                                     </listBibl>
+
+                             </source>
+                          </recordHist>
+                       </adminInfo>
+                    </additional>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                     href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+          </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <particDesc>
+               <listPerson>
+                  <person>
+                     <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                  </person>
+                  <person>
+                     <persName ref="PRS1071dAbbadi" role="owner"/>
+                     </person>
+                </listPerson>
+                </particDesc>
+            <textClass>
+                <keywords>
+                    <term key="Bible"/>
+                    <term key="OldTestament"/>
+                    <term key="Religion"/>
+                    <term key="Gon"/>
+                    </keywords>
```

Comment:
if it is a manuscript belonging to the Christian tradition, ChristianLiterature should be added

Reply:
Done.